sdcm/tester.py

```diff
@@ -149,6 +149,10 @@ def init_resources(self, n_db_nodes=None, n_loader_nodes=None, dbs_block_device_
             n_db_nodes = self.params.get('n_db_nodes')
         if n_loader_nodes is None:
             n_loader_nodes = self.params.get('n_loaders')
+        if n_loader_nodes <= 0:
+            self.error('Invalid parameter n_loader_nodes/n_loaders: %s,'
+                       ' it should be larger than zero.' % n_loader_nodes)
```

Comment:
If we set n_load_nodes to zero, the error will be raised after costing long time on init db nodes.
So it's worth to add this check. If we set an invalid n_db_nodes, the error will be raised very early.

However, it's find to add meaningful error for invalid n_db_nodes. I'll add a similar check for n_db_nodes.